src/main/resources/application.yaml

```diff
@@ -55,6 +55,11 @@ darts-gateway:
   register-node:
     schema: "src/main/resources/schemas/darts-register-node.xsd"
     validate: false
+  add-audio:
+    schema: "src/main/resources/schemas/addAudio.xsd"
+    validate: false
+    # Represents 360 MB per channel as maximum upload
+    fileSizeInBytes: 377487360
```

Comment:
I noticed earlier today @cmagowan-hmcts changed max-file-size: 350MB
I'd expect it needs to be bumped to be 360MB in the darts-api with the comment changed / removed.

https://github.com/hmcts/darts-api/blob/master/src/main/resources/application.yaml#L68 
```
  servlet:
    multipart:
      max-file-size: 350MB #this affects addAudio as well, so needs to handle a 350mb mp2 file
      max-request-size: 360MB #this affects addAudio as well, so needs to handle a 350mb mp2 file
```

Reply:
Fair point. I'll chat to Chris. One of the chats mentioned a 354MB file per channel as a max so 350 probably isn't gonna cut it